htlcswitch/link_test.go

```diff
@@ -386,8 +383,7 @@ func TestChannelLinkSigThenRev(t *testing.T) {
 	require.True(t, ok)
 
 	// Restart Alice so she sends and accepts ChannelReestablish.
-	cleanUp = alice.restart(false, true)
-	defer cleanUp()
+	alice.restart(t, false, true)
```

Comment:
No need to add `t` to `restart`, the harness already has `h.t` that can be used.